test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -300,7 +300,11 @@ string TestFunctionCall::formatRawParameters(
 		{
 			if (param.format.newline)
 				os << endl << _linePrefix << "// ";
-			os << param.rawString;
+			for (auto const c: param.rawString)
+				if (c >= ' ')
+					os << c;
+				else
+					os << "\\x" << solidity::util::toHex(static_cast<uint8_t>(c));
```

Comment:
Maybe something like:
```suggestion
			for (auto const c: param.rawString)
				os << (c >= ' ' ? string(1, c) : "\\x" + solidity::util::toHex(static_cast<uint8_t>(c)));
```